app/scripts/modules/core/src/managed/ManagedWriter.ts

```diff
@@ -88,6 +90,7 @@ export class ManagedWriter {
       .one('constraint')
       .post({
         type,
+        artifactReference: reference,
         artifactVersion: version,
```

Comment:
it'd be nice if we could rename these fields to `reference` and `version` to match all the other endpoints in keel — it's easy to spot that this particular feature was implemented by a different human at a different time from all the others.

Reply:
:D totally